slack-lint/src/main/java/slack/lint/DeprecatedAnnotationDetector.kt

```diff
@@ -16,13 +16,19 @@
 package slack.lint
 
 import com.android.tools.lint.client.api.LintClient
-import com.android.tools.lint.detector.api.Category
-import com.android.tools.lint.detector.api.Implementation
-import com.android.tools.lint.detector.api.Issue
-import com.android.tools.lint.detector.api.Severity
+import com.android.tools.lint.detector.api.*
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiMethod
+import org.jetbrains.uast.UAnnotation
+import org.jetbrains.uast.UElement
 import slack.lint.util.Priorities
 import slack.lint.util.sourceImplementation
 
+private const val DEPRECATED_ANNOTATION_NAME_JAVA = "java.lang.Deprecated"
+private const val DEPRECATED_ANNOTATION_NAME_KOTLIN = "kotlin.Deprecated"
+private const val BRIEF_DESCTIPIOTN_PREFIX_DEFAULT = "This class or method"
```

Comment:
I don't know how to pronounce `DESCTIPIOTN`

Reply:
It was late by the time I finally got this working.